### ISSUE TITLE
A11y improvements

### ISSUE THREAD

**Issue by JohannesDoberer** [luigi-project/luigi#4584]
- unify key up event and use `event.key === Enter` instead of `event.code === 'Enter'`
- tbd

**Comment by walmazacn** [comment]
Lighthouse report issues:
- [x] ARIA attributes do not match their roles
- [x] Buttons do not have an accessible name
- [x] HTML element does not have lang attribute
- [x] IFRAME elements do not have a title
